setup_root.md

```diff
@@ -1,65 +1,211 @@
+    * The default zendro username is *zendro-admin* and the password is *admin*.
+
+  ![spa example](figures/login.png)
+  ![spa example](figures/spa.png)
+
+* GraphQL API: http://localhost:3000/graphql
+
+  ![api example](figures/graphql.png )
+
+* GraphQL API with authenthication: http://localhost:7000
+    * The default zendro username is *zendro-admin* and the password is *admin*.
+
+  ![api example](figures/login.png)
+  ![api example](figures/graphiql.png)
+
+
+If you wish to modify the default ports, adjust next files:
+> [Stop Zendro Instance](#stop), modify files and [start Zendro Instance](#start) again.
+
+* ./docker-compose-prod.yml
+* ./docker-compose-dev.yml
+* ./single-page-app/.env.production
+* ./single-page-app/.env.development
+* ./single-page-app/package.json
+* ./graphql-server/.env
+* ./graphiql-auth/.env.development
+* ./graphiql-auth/.env.production
+* ./graphiql-auth/package.json
+
+Also, if you wish to modify docker containers name or docker services names, adjust next files:
+> [Stop Zendro Instance](#stop), modify files and [start Zendro Instance](#start) again.
+
+* ./docker-compose-prod.yml
+* ./docker-compose-dev.yml
+
+Moreover, if you wish to modify keycloak IP adjust:
+* ipv4_address
+* subnet
+
+in *./docker-compose-prod.yml* and *./docker-compose-dev.yml*.
+
+
+#### Without docker
+If you prefer to use local setup with Keycloak, there are a few things to do after running Zendro:
+
+***MISSING***
+
+* install java
+* Install keycloak
+* Env vars and where to find them in keycloak, screenshots.
+* logs
+
+
+### Step 7: Start up Zendro with access control 
+Zendro can be used checking access rights for every single GraphQL query received by the currently logged in user identified by the Token. The user is decoded and his roles are loaded to check his access rights. This step is carried out by the [NPM acl package](https://www.npmjs.com/package/acl). Respective access rights can and must be declared in the file `./graphql-server/acl_rules.js`.
+
+You can run Zendro with or without this access control check. The default is to run it without checking access rights.
+
+To switch access right check on, you must uncomment the command line switch acl and change the following line in `./graphql-server/migrateDbAndStartServer.sh`
+
+```
+npm start #acl
+```
+to 
+```
+npm start acl
+```
+
+Moreover, if you decide not to use access control, you can restrict the user permissions using the enviroment variable *WHITELIST_ROLES*. For example, if you wish users only can do reading querys, in `./graphql-server/.env` file modify:
+
+```
+WHITELIST_ROLES="reader"
+```
+
+You can add all roles you wish separating them with a comma.
+
+
+<!----><a name="stop"></a>
+### Step 8: Stop Zendro instance
+Execute the next command to stop Zendro and remove all volumes.
+```
+$ zendro dockerize -d -v
+```
+
+If you are on production mode execute:
+```
+$ zendro dockerize -d -p -v
+```
+
+
+
+### Customize your project (optional)
```

Comment:
`Furthermore the whole codebase used to run zendro is exposed an can be directly customized if needed.`
Furthermore, ... exposed and ....